ui/raidboss/data/06-ew/trial/golbez-ex.txt

```diff
@@ -1,6 +1,372 @@
+461.4 "Void Stardust"
+
+# Mini-phase 2, exaflares -> pairs -> role stacks or light parties -> out then light parties
+
+500.0 "Azdaja's Shadow" sync / 1[56]:[^:]*:Golbez:8478:/
+505.2 "Flames of Eventide" sync / 1[56]:[^:]*:Golbez:847B:/
+508.3 "Flames of Eventide" sync / 1[56]:[^:]*:Golbez:847B:/
+511.4 "Flames of Eventide" sync / 1[56]:[^:]*:Golbez:847B:/
+513.4 "--sync--" sync / 1[56]:[^:]*:Golbez:84B8:/
+518.5 "Void Stardust (preview)" sync / 1[56]:[^:]*:Golbez:84A4:/
+524.6 "Void Stardust" duration 2.3 sync / 1[56]:[^:]*:Golbez:84A6:/
+530.4 "Abyssal Quasar" sync / 1[56]:[^:]*:Golbez:84AB:/
+535.4 "Eventide Triad/Fall" sync / 1[56]:[^:]*:Golbez:848[05]:/
+548.5 "Binding Cold" sync / 1[56]:[^:]*:Golbez:84B3:/
+
+556.6 "Void Meteor" sync / 1[56]:[^:]*:Golbez:84AD:/
+556.7 "Void Comet x4" duration 3 #sync / 1[56]:[^:]*:Golbez:84B0:/
+560.7 "Void Meteor" sync / 1[56]:[^:]*:Golbez:84B1:/
+
+562.7 "--sync--" sync / 1[56]:[^:]*:Golbez:84B8:/
+570.0 "Phases of the Shadow (cast)" sync / 1[56]:[^:]*:Golbez:86E7:/
+571.8 "Rising Beacon (record)" sync / 1[56]:[^:]*:Shadow Dragon:848D:/
+571.8 "Phases of the Shadow (front)" sync / 1[56]:[^:]*:Golbez:86E9:/
+573.5 "Phases of the Shadow (back)" sync / 1[56]:[^:]*:Golbez:86EF:/
+574.7 "Rising Beacon (out)" sync / 1[56]:[^:]*:Golbez:86EC:/
+576.8 "--sync--" sync / 1[56]:[^:]*:Golbez:8475:/
+577.2 "Immolating Shade (light parties)" sync / 1[56]:[^:]*:Golbez:8496:/ jump 900.0
+
+# timeline rolling
+
+584.1 "Terrastorm"
+590.2 "Arctic Assault"
+591.2 "Arctic Assault"
+593.2 "Void Blizzard III"
+593.2 "Terrastorm"
+600.3 "Binding Cold"
+
+# Mini-phase 2, exaflares -> pairs -> role stacks or light parties -> in then spread
+
+700.0 "Azdaja's Shadow" sync / 1[56]:[^:]*:Golbez:8479:/
+705.2 "Flames of Eventide" sync / 1[56]:[^:]*:Golbez:847B:/
+708.3 "Flames of Eventide" sync / 1[56]:[^:]*:Golbez:847B:/
+711.4 "Flames of Eventide" sync / 1[56]:[^:]*:Golbez:847B:/
+713.4 "--sync--" sync / 1[56]:[^:]*:Golbez:84B8:/
+718.5 "Void Stardust (preview)" sync / 1[56]:[^:]*:Golbez:84A4:/
+724.6 "Void Stardust" duration 2.3 sync / 1[56]:[^:]*:Golbez:84A6:/
+730.4 "Abyssal Quasar" sync / 1[56]:[^:]*:Golbez:84AB:/
+735.4 "Eventide Triad/Fall" sync / 1[56]:[^:]*:Golbez:848[05]:/
+748.5 "Binding Cold" sync / 1[56]:[^:]*:Golbez:84B3:/
+
+756.6 "Void Meteor" sync / 1[56]:[^:]*:Golbez:84AD:/
+756.7 "Void Comet x4" duration 3 #sync / 1[56]:[^:]*:Golbez:84B0:/
+760.7 "Void Meteor" sync / 1[56]:[^:]*:Golbez:84B1:/
+
+762.7 "--sync--" sync / 1[56]:[^:]*:Golbez:84B8:/
+770.0 "Phases of the Shadow (cast)" sync / 1[56]:[^:]*:Golbez:86E7:/
+771.8 "Rising Ring (record)" sync / 1[56]:[^:]*:Shadow Dragon:848E:/
+771.8 "Phases of the Shadow (front)" sync / 1[56]:[^:]*:Golbez:86E9:/
+773.5 "Phases of the Shadow (back)" sync / 1[56]:[^:]*:Golbez:86EF:/
+774.7 "Rising Ring (under)" sync / 1[56]:[^:]*:Golbez:86ED:/
+776.8 "--sync--" sync / 1[56]:[^:]*:Golbez:8475:/
+777.2 "Immolating Shade (spread)" sync / 1[56]:[^:]*:Golbez:8494:/ jump 900.0
```

Comment:
900? (same question for the other 900 jump above) Ideally jumps go to existing lines (and probably I should write some test for that, at least in cactbot), but I'm wondering if this is an error.  Did you run test_timeline against this?

Reply:
Yeah, I adjusted the timeline up to 1000 and forgot to bump this jump.